### PR TITLE
Remove softfailure for bsc#1192992

### PR DESCRIPTION
**Pull request by hjluo**
- Description:
      -  Remove soft failure for bsc#1192992
- Related ticket: 
     - https://progress.opensuse.org/issues/151849
- Needles: 
     - N/A
- Verification run: 
     - [**VR**](https://openqa.suse.de/tests/overview?build=hjluo%2Fos-autoinst-distri-opensuse%23first_boot_workaround&distri=sle&version=15-SP6)

### CODE REVIEW

File: lib/x11utils.pm [hunk 1]
```diff
@@ -229,12 +229,6 @@ sub handle_additional_polkit_windows {
         # for S390x testing, since they are not using qemu built-in vnc, it is
         # expected that polkit authentication window can open for first time login.
         # see bsc#1177446 for more information.
-        # Base latest feedback of bsc#1192992,authentication should never open if is_sle  >= 15SP4
-        if (is_sle('>=15-sp4')) {
-            record_soft_failure 'bsc#1192992 - authentication should never open if is_sle >= 15SP4';
-        } else {
-            record_info('authentication open for first time login');
```
[the remark this answers is not in the document]
Comment (hjluo) [reply lib/x11utils.pm:236]: > Why is that `record_info` also removed? Don't we need it to understand what is happening?

Added it back && thank you.

File: lib/x11utils.pm [hunk 2]
```diff
@@ -229,12 +229,7 @@ sub handle_additional_polkit_windows {
...
-        }
+        record_info('authentication open for first time login');
```
Comment (jknphy) [inline lib/x11utils.pm:232]: please add the bug reference in the message, it is a strange behavior, better overexplain it.

**Review by jknphy — APPROVED** [review]

File: lib/x11utils.pm (hunk 2 above)
Comment (hjluo) [reply lib/x11utils.pm]: changed && thank you.

**Review by rakoenig — APPROVED** [review]
LGTM.

**Pull request merged**